src/SDKs/Batch/DataPlane/Azure.Batch.Unit.Tests/ComputeNodeUnitTests.cs

```diff
@@ -6,13 +6,17 @@
     using System;
     using System.Collections.Generic;
     using System.Linq;
+    using System.Threading.Tasks;
     using BatchTestCommon;
     using Microsoft.Azure.Batch;
     using Microsoft.Azure.Batch.Common;
+    using Microsoft.Azure.Batch.Auth;
+    using Microsoft.Rest.Azure;
     using TestUtilities;
     using Xunit;
     using Xunit.Abstractions;
     using Protocol=Microsoft.Azure.Batch.Protocol;
+    using Models = Microsoft.Azure.Batch.Protocol.Models;
```

Comment:
Looks like they were added by @smith1511  - not sure why. Maybe he wrote his tests here initially and then move them to their own file and forgot to remove these?

In any case I've fixed it.